index.md

```diff
@@ -163,6 +163,7 @@ customization safe and easy.
 
 ### Tutorials
 
+* [Abdullah](https://abdullah.today/about.html){:target="_blank" rel="noopener"} wrote [how to secure your dotfiles](https://abdullah.today/2019/10/01/encrypted-dotfiles.html){:target="_blank" rel="noopener"} with keybase.
```

Comment:
The `{:target="_blank" rel="noopener"}` shows up visible in the rendered Markdown - please remove it and just use the standard markdown link syntax.

Thanks!